### PR TITLE
added information for install on older PHP version

### PR DESCRIPTION
**Pull request by grogy**
Hi,

I am trying how add this tool to project with older PHP. Here is described setup. What do you think about it?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,21 @@ At the moment these packages are supported:
 composer require rector/rector --dev
 ```
 
+# Install on PHP < 7.1
+
+You must have separated environment with PHP 7.1 (for example in Docker container). When you have it then run next command. The reason for this difficulty setup is your dependencies are only PHP <7.1 compatible and you cannot add 7.1.x package.
```
Comment (TomasVotruba) [inline README.md]: - run next => run following
- "The reason for this difficulty setup is your dependencies are only PHP <7.1 compatible and you cannot add 7.1.x package." 
   - I think this information is already included in the title.

File: README.md [hunk 2]
```diff
@@ -21,6 +21,21 @@ At the moment these packages are supported:
...
+
+```
+composer create-project rector/rector your-path-to-rector
```
Comment (TomasVotruba) [inline README.md]: I like this!

File: README.md [hunk 3]
```diff
@@ -21,6 +21,21 @@ At the moment these packages are supported:
...
+```
+
+When do you have it then you can run all commands like
+
+```
+your-path-to-rector/bin/rector reconstruct src --level=nette24
+your-path-to-rector/bin/rector reconstruct src --level=symfony40
```
Comment (TomasVotruba) [inline README.md]: In this case maybe more common use case will be: 

```bash
your-path-to-rector/bin/rector reconstruct /var/www/old-project --level=nette24
your-path-to-rector/bin/rector reconstruct /var/www/another-old-project --level=symfony40
```

In words: reconstruct more code projects that are not co-located with Rector directory.

File: README.md (hunk 1 above)
Comment (grogy) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (grogy) [reply README.md]: Fixed

**Comment by TomasVotruba** [comment]
Thank you, merged :+1:

**Pull request merged**